### ISSUE TITLE
关于qwen3-vl 微调时视频数据集的宽高resize 问题

### ISSUE THREAD

**Issue by Key-lei** [QwenLM/Qwen3-VL#1786]
期待回复 ：）

具体问题如下：
（1）当我希望能够固定视频中每一帧的size的时候，我试图通过在message中加入
```
messages = [
    {
        "role": "user",
        "content": [
            {
                "type": "video",
                "video": "video_url",
                "total_pixels":24 * 448 * 448,
                "min_pixels":448*448,
                "max_frames": 24,
                "sample_fps":2,
                "resized_height": 448,
                "resized_width": 448,
            },
            {"type": "text", "text": "Describe this video."},
        ],
    }
]
```
最后经过
```
inputs_1 = processor.apply_chat_template(
    messages,
    tokenize=True,
    add_generation_prompt=True,
    return_dict=True,
    return_tensors="pt"
)
```
的输出如下：
```
{'input_ids': tensor([[151644,   8948,    198,  ..., 151644,  77091,    198]]), 'attention_mask': tensor([[1, 1, 1,  ..., 1, 1, 1]]), 'pixel_values_videos': tensor([[-1.0623, -1.4857, -1.6317,  ..., -0.7834, -0.7123, -0.6981],
        [-1.6025, -1.5295, -1.3105,  ..., -0.2573, -0.3000, -0.3711],
        [-1.6755, -1.6901, -1.7193,  ..., -0.6128, -0.5275, -0.4706],
        ...,
        [-1.6025, -1.6171, -1.6317,  ..., -1.2527, -1.2811, -1.2811],
        [ 0.9230, -0.8872, -1.6171,  ..., -1.1958, -1.1958, -1.1958],
        [ 0.7333, -1.7485, -1.7047,  ..., -1.3665, -1.3807, -1.3807]]), 'video_grid_thw': tensor([[230,  40,  72]]), 'video_second_per_grid': tensor([1.])}
```
我理解其size 仍然是 640 * 1152 我的原始视频尺寸为 1080 * 1920 。是我参数设置不对吗？希望得到解答，同样的resize方法在qwen2_5 vl 或者omni 中是正常生效的。

**Comment by JJJYmmm** [comment]
`processor.apply_chat_template`走的是hf的实现，不支持resized_height/resized_width, 只支持smart resize. 

`同样的resize方法在qwen2_5 vl 或者omni 中是正常生效的。` 这句话应该指的是使用qwen-vl-utils？vlutils是支持resized_height/resized_width的。

**Comment by Key-lei** [comment]
> `processor.apply_chat_template`走的是hf的实现，不支持resized_height/resized_width, 只支持smart resize.
> 
> `同样的resize方法在qwen2_5 vl 或者omni 中是正常生效的。` 这句话应该指的是使用qwen-vl-utils？vlutils是支持resized_height/resized_width的。

感谢您的回复，我理解了，谢谢。

**Comment by wuaman** [comment]
> `processor.apply_chat_template`走的是hf的实现，不支持resized_height/resized_width, 只支持smart resize.
> 
> `同样的resize方法在qwen2_5 vl 或者omni 中是正常生效的。` 这句话应该指的是使用qwen-vl-utils？vlutils是支持resized_height/resized_width的。

您好，我在用qwen2.5-vl微调OCR任务，用的是hf的peft进行lora微调。message构建格式如下：

	messages = [
		{
			'role': 'user',
			'content': [
				{
					'type': 'image',
					'image': file_path,
					# 'resized_height': 280,
					# 'resized_width': 280,
				},
				{'type': 'text', 'text': instruction_text},
			],
		}
	]


	text = processor.apply_chat_template(messages, tokenize=False, add_generation_prompt=True)
	image_inputs, video_inputs = process_vision_info(messages) # process_vision_info 是qwen_vl_utils里面的

	inputs = processor(
		text=[text],
		images=image_inputs,
		videos=video_inputs,
		padding=True,
		return_tensors='pt',
	)
请问我需要对图像做resize处理吗？
如果需要的话，**resize**的尺寸是多少？ 是在上述构建messages时添加 **resized_width/resized_height**  参数还是在其他地方做数据预处理呢？
ps：我的图片是经过bbox裁切出来的，像素尺寸普遍不大 ( 类似[LaTeX_OCR](https://modelscope.cn/datasets/AI-ModelScope/LaTeX_OCR) )

期待您的回复，感谢
